packages/x-components/src/views/home/Home.vue

```diff
@@ -363,7 +371,7 @@
                               :max-items-to-render="3"
                             >
                               <NextQuery
-                                #default="{ suggestion: nextQuery }"
+                                v-slot="{ suggestion: nextQuery }"
```

Comment:
I guess it was auto-fixed by eslint, we should keep with `#default`.
```suggestion
                                #default="{ suggestion: nextQuery }"
```